release_bot/configuration.py

```diff
@@ -179,5 +185,54 @@ def get_pypi_project_from_setup_cfg(setup_cfg):
 
         return None
 
+    def get_project(self):
+        """
+        Create ogr lib project instance based on provided configuration.
+        Project instance is used for manipulating with Github/Pagure repo.
+        :return: ogr Github/Pagure project instance or None
+        """
+        if self.github_token != '' and self.github_username != '':
+            return self.get_github_project(username=self.github_username,
+                                           repository=self.repository_name,
+                                           token=self.github_token)
+        elif self.pagure_token != '' and self.pagure_username != '':
+            return self.get_pagure_project(username=self.pagure_username,
+                                           repository=self.repository_name,
+                                           token=self.pagure_token)
```

Comment:
[This PR](https://github.com/packit-service/ogr/pull/95) should help you with that..;-) Feel free to add some comments. I would appreciate some opinions from the perspective of release-bot.

We can release a new version of OGR after merging that...

Reply:
My current solution is easier for required options `github_token` and `github_username` in `conf.yaml`.  Your [PR](https://github.com/packit-service/ogr/pull/95) creates ogr project instance from url. I can use it, but first, we need to discuss #230. Option `clone_url` in `conf.yaml` must be used. 

Sorry, I missed adding a comment to your [PR](https://github.com/packit-service/ogr/pull/95).

Reply:
We can discuss that in #230 ...